ci-operator/config/openshift/verification-tests/openshift-verification-tests-main__ota-multi-stable-4.19-upgrade-from-stable-4.18.yaml

```diff
@@ -85,6 +90,32 @@ tests:
     - ref: cucushift-upgrade-healthcheck
     - ref: cucushift-ota-postupgrade
     workflow: cucushift-installer-rehearse-azure-ipi
+- as: baremetal-ipi-retarget-f28
```

Comment:
is y-then-y still needed?

Reply:
I tried y-then-y, but 4.20 image will cause all jobs in a config file broken.

Reply:
I see y-then-y and z-then-y are still listed in the pr's description. If you intend to drop them in this pr, please update it accordingly to reflect the real test coverage. 
But I guess y-retarget is an important part for OTA-861, maybe we can split it to two cases, current ocp-25473 together with the pr is for z-retarget, and another case and following pr is for y-retarget once 4.20 is available.

Reply:
done